context/app/static/js/components/entity-search/searchkit-modifications/withSearchkitRouting.js

```diff
@@ -0,0 +1,166 @@
+// Copied from https://github.com/searchkit/searchkit/blob/6d11b204520009a705fe207535bd4f18d083d361/packages/searchkit-client/src/withSearchkitRouting.tsx
+// Modified to handle custom route state
```

Comment:
Could you had a comment bracketing the lines that have changed?

Reply:
There are no meaningful changes in this PR outside of eslint fixes and removing typescript. The comment shown was made in advance of changes in the future PRs, I'll add comments there.